OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -214,65 +9135,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            target="#65ra"/>) </item>
+                                </list>
                             </handNote>
                         </handDesc>
-                        
-                       <!-- <decoDesc>
+
+                        <!-- <decoDesc>
                          
                         </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
+
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus from="1v" to="2r"/>
+                                    <desc>Several pen trials, one of which contains the name of <persName ref="PRS13094GabraKr"/>.
+                                    Three longer pen trials contain the beginning of the Sǝnkǝssār, excerpts from the Psalms of David and
+                                    a liturgical prayer.
+                                    </desc>
+                                    <q xml:lang="gez"/>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#3v #21r"/>
+                                    <desc>Scribbles or pen trials.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#4r #79r #109r"/>
+                                    <desc> The decorated names of the months Maskaram, Ḫǝdār and
+                                        Tāḫśāś are written in the upper margin.</desc>
+                                </item>
+
+                                <item xml:id="e4">
+                                    <locus
+                                        target="#4r #14r #24r #34r #44r #56r #68r #78r #79r #88r #96 #104r #109r
+                                        #118v #119r #129r #139r #147r #157r #167r #175r #184r #194r #204r"/>
+                                    <desc>The word <foreign xml:lang="gez">ጥራዝ፡</foreign>, ‘quire’,
+                                        written in red or black and decorated with dots and lines,
+                                        is written in the lower margin (eg., <locus target="#4r"/>)
+                                        or both in the lower and upper margin of the first recto of
+                                        all quires, except for the protective quires. It is also
+                                        written on <locus target="#79r"/> and <locus target="#118v"
+                                        />, the second and last page of a quire respectively. On
+                                            <locus target="#4r"/> and <locus target="#129r"/>, it is
+                                        followed by the name of the scribe, <foreign xml:lang="gez"
```

Comment:
That is a very interesting feature!! It may indicate that he did not work alone but in a scriptorium. I would definitely take this also under hands or somewhere else. I think, at least in this case, the reader would like to have the information summarized somewhere. 
In addition, I would note this rare scribal practise  in the introduction as a particluar feature.